src/commands/cmd_call.c

```diff
@@ -0,0 +1,35 @@
+/*
+* Copyright 2018-2019 Redis Labs Ltd. and Contributors
+*
+* This file is available under the Apache License, Version 2.0,
+* modified with the Commons Clause restriction.
+*/
+
+#include "cmd_call.h"
+#include "../algorithms/algorithms.h"
+
+#include <stdlib.h>
+#include <assert.h>
+
+int MGraph_Call(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
+    if (argc != 2) return RedisModule_WrongArity(ctx);
```

Comment:
I think `argc < 2` should be the failure case; many algorithms will have user-provided arguments.